website/content/docs/platform/mssql/troubleshooting.mdx

```diff
@@ -19,11 +19,37 @@ your issue, you can [enable trace logging](/vault/docs/platform/mssql/configurat
 Restart SQL Server for the config change to take effect, and you should see more
 detailed logs in the same section of Windows Event Viewer.
 
+## Check SQL Server error logs
+
+If the Vault EKM provider is not generating event logs, you may find some
+information in the SQL Server's error logs. Check for an event with ID
+17111 from "MSSQLSERVER" which details the file location of the error logs,
+e.g. "Logging SQL Server messages in file
+'C:\Program Files\Microsoft SQL Server\MSSQL15.MSSQLSERVER\MSSQL\Log\ERRORLOG'."
+
+## Authenticode error
+
+When running the `CREATE CRYPTOGRAPHIC PROVIDER` SQL query, if you get an error
+"Cannot load library", you may see "Failed to verify Authenticode signature on
+DLL" in the error logs. This can normally be remediated by updating the CA trust
+store to include the latest Digicert certificates. Run the following to import
+the required certificates into the Root trust store:
+
+```powershell
+New-Item -ItemType Directory -Path certs
+certutil -syncwithWU .\certs\
+extrac32 -L .\certs\ .\certs\authrootstl.cab
+certutil -f -ent -AddStore Root .\certs\authroot.stl
+certutil -f -ent -AddStore Root .\certs\0563b8630d62d75abbc8ab1e4bdfb5a899b24d43.crt
```

Comment:
I assume these are example cert names. If so, we should add a note before "Run the following..." to also replace the cert names with actual values.

Reply:
They're actually the exact Digitcert ones required :) the -syncwithWU command above downloads dozens of certs.